In [1]:
# Fetching data with the block height from whatsonchain
import requests
api_url = 'https://api.whatsonchain.com/v1/bsv/main/block/height/735710'
result = requests.get(api_url)

In [3]:
#Getting data
d = result.text

In [6]:
#importing and loading in json
import json
j_data = json.loads(d)

In [7]:
#Time
j_data['time']

1650293886

In [8]:
#Total Fees
j_data['totalFees']

0.07741249000000039

In [9]:
#txcount
j_data['txcount']

3101

In [10]:
#Confirmations
j_data['confirmations']

1000

In [11]:
#miner
j_data['miner']

'qdlnk'

In [12]:
#time conversion to human readable format
from datetime import datetime
ts = int('1650293886')
print(datetime.utcfromtimestamp(ts).strftime('%Y-%m-%d %H:%M:%S'))

2022-04-18 14:58:06


1.txcount= It is the transaction number and every transaction number is unique. The transaction number is 3101 in this particular transaction
2.time= This is transaction time. And the time is exact according to when the transaction is being made. Like here, the transaction time is 2022-04-18 14:58:06.
3.totalFees= This shows the amount that is charged when the transaction is being made. Here the transaction fee is 0.0774125
4.confirmation= Confirmation is number of blocks in the block chain that is accepted by the network meaning the transaction that is made is accepted and is valid.
5.miner= Miners are the one who verifies the transaction first. Here qdlink is miner for this particular transaction.